airmozilla/main/templates/main/calendar.html

```diff
@@ -29,6 +30,8 @@
   <h2 class="section-title">Calendar</h2>
   <div id="calendar" data-events-url="{{ url('main:calendar_data') }}"></div>
 
+  <input type="checkbox"> Week starts on Monday.</input>
```

Comment:
That's not how you do a checkbox input. It needs to have an id and a label so that if you click the label in toggles the checkbox. Also, an input tag is not a block element so it needs to be wrapped in one. Try something like this:
```html
<p>
  <input type="checkbox" id="id_startsonmonday">
 <label for="id_startsonmonday">Week starts on a Monday</label>
</p>
```

The `label` tag might have some CSS inherited that you might need to wrangle.